guide/guide-measurement.rst

```diff
@@ -176,27 +235,65 @@ distribution of the outcomes exactly in a single line:
+   right (`1.0`) eigenstates.
+
+ All three lists are in the same order -- i.e. the first eigenvalue is
+ `eigenvalues[0]`, its corresponding eigenstate is `eigenstates[0]`, and
+ its probability is `probabilities[0]`, and so on.
+
+Similarly, when we want to measure using projection operators, we can use the
+`measurement_statistics` functions. Consider again, the state :math:`\ket{0+}`.
+Suppose, now we want to obtain the measurement outcomes for the second qubit. We
+must use the projectors specified earlier by `PZ2` which allow us to measure only
+on the second qubit. Since the second qubit has the state :math:`\ket{+}`, we get
+the following result.
+
+.. ipython::
+
+   In [1]: collapsed_states, probabilities = measurement_statistics(PZ2, state_0plus)
+
+   In [2]: collapsed_states
+   Out[2]: [Quantum object: dims = [[2, 2], [1, 1]], shape = (4, 1), type = ket
+            Qobj data =
+            [[1.]
+            [0.]
+            [0.]
+            [0.]], Quantum object: dims = [[2, 2], [1, 1]], shape = (4, 1), type = ket
+            Qobj data =
+            [[0.]
+            [1.]
+            [0.]
+            [0.]]]
+
+   In [3]: probabilities
+   Out[3]: [0.4999999999999999, 0.4999999999999999]
+
+The :func:`~qutip.measurement.measurement_statistics` function returns two values:
 
-* `eigenvalues` is an array of eigenvalues of the measurement operator, i.e.
-  a list of the possible measurement results. In our example
-  the value is `array([-1., -1.])`.
+* `collapsed_states` is an array of the possible final states after the
+  measurement is complete. Each element of the array is a :obj:`~qutip.Qobj`.
 
-* `eigenstates` is an array of the eigenstates of the measurement operator, i.e.
-  a list of the possible final states after the measurement is complete.
-  Each element of the array is a :obj:`~qutip.Qobj`.
+* `probabilities` is a list of the probabilities of each measurement outcome.
 
-* `probabilities` is a list of the probabilities of each measurement result.
-  In our example the value is `[0.5, 0.5]` since the `up` state has equal
-  probability of being measured to be in the left (`-1.0`) or
-  right (`1.0`) eigenstates.
+Note that the collapsed_states are exactly :math:`\ket{00}` and :math:`\ket{01}`
+with equal probability, as expected. The two lists are in the same order.
 
-All three lists are in the same order -- i.e. the first eigenvalue is
-`eigenvalues[0]`, its corresponding eigenstate is `eigenstates[0]`, and
-its probability is `probabilities[0]`, and so on.
 
 The `measurement_statistics` function can provide statistics for measurements
 of density matrices too. In this case `projectors` from the density matrix
 onto the corresponding `eigenstates` are returned instead of the `eigenstates`.
 You can read about these and other details at
-:func:`~qutip.measurement.measurement_statistics`.
+:func:`~qutip.measurement.measurement_statistics` and :func:`~qutip.measurement.measurement_statistics_observable`.
```

Comment:
Just curious, what does the `~` means here in `~qutip.measurement.measurement_statistics`?

Reply:
@BoxiLi the tilde is used by intersphinx to find the object and create an hyperlink, without showing the qutip prefix.